python/cog/server/http.py

```diff
@@ -359,12 +375,15 @@ def _signal_set_event(signum: Any, frame: Any) -> None:
 
     config = load_config()
 
-    threads = args.threads
+    threads: Optional[int] = args.threads
     if threads is None:
         if config.get("build", {}).get("gpu", False):
             threads = 1
         else:
             threads = os.cpu_count()
+            if threads is None:
+                log.warn("Unable to determine cpu count, defaulting to 1 thread")
+                threads = 1
```

Comment:
in general os.sched_getaffinity(0) would be better for this but it doesn't actually matter in the slightest because this isn't used in practice and doesn't allow any degree of concurrency